ps2.xml

```diff
@@ -2457,6 +2457,13 @@
                     <name>D_STADR</name>
                     <description>DMAC stall address</description>
                     <addressOffset>0x6060</addressOffset>
+                    <fields>
+                        <field>
+                            <name>ADDR</name>
+                            <description>Stall address (qword alignment)</description>
+                            <bitRange>[30:0]</bitRange>
```

Comment:
If this field is qword-aligned...wouldn't it make sense to truncate the bottom bits, too?

Reply:
I was just typing something about that, I was going to say they're always meant to be zero so I could truncate them but then as they're always addresses maybe the numbers should be kept to the same magnitude